molecule/packages-cluster/group_vars/all

```diff
@@ -64,6 +64,7 @@ pulp_settings:
       NAME: pulp
       USER: pulp
       PASSWORD: password
+  telemetry: false
```

Comment:
Put this in https://github.com/pulp/pulp_installer/blob/main/molecule/scenario_resources/group_vars/all instead of all these individual files.